unixfs/src/pb/unixfs.rs

```diff
@@ -0,0 +1,229 @@
+// Modified automatically generated rust module for 'unixfs.proto' file
+#![allow(rust_2018_idioms)]
+#![allow(non_snake_case)]
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(unused_imports)]
+#![allow(unknown_lints)]
+#![allow(clippy::all)]
+#![cfg_attr(rustfmt, rustfmt_skip)]
+use super::*;
+use quick_protobuf::sizeofs::*;
+use quick_protobuf::{BytesReader, MessageRead, MessageWrite, Result, Writer, WriterBackend};
+use std::borrow::Cow;
+use std::convert::TryFrom;
+use std::io::Write;
+use std::ops::Deref;
+use std::ops::DerefMut;
+#[derive(Debug, Default, PartialEq, Clone)]
+pub struct Data<'a> {
+    pub Type: mod_Data::DataType,
+    pub Data: Option<Cow<'a, [u8]>>,
+    pub filesize: Option<u64>,
+    pub blocksizes: Vec<u64>,
+    pub hashType: Option<u64>,
+    pub fanout: Option<u64>,
+    pub mode: Option<u32>,
+    pub mtime: Option<UnixTime>,
+}
+impl<'a> MessageRead<'a> for Data<'a> {
+    fn from_reader(r: &mut BytesReader, bytes: &'a [u8]) -> Result<Self> {
+        let mut msg = Self::default();
+        while !r.is_eof() {
+            match r.next_tag(bytes) {
+                Ok(8) => msg.Type = r.read_enum(bytes)?,
+                Ok(18) => msg.Data = Some(r.read_bytes(bytes).map(Cow::Borrowed)?),
+                Ok(24) => msg.filesize = Some(r.read_uint64(bytes)?),
+                Ok(32) => msg.blocksizes.push(r.read_uint64(bytes)?),
+                Ok(40) => msg.hashType = Some(r.read_uint64(bytes)?),
+                Ok(48) => msg.fanout = Some(r.read_uint64(bytes)?),
+                Ok(56) => msg.mode = Some(r.read_uint32(bytes)?),
+                Ok(66) => msg.mtime = Some(r.read_message::<UnixTime>(bytes)?),
+                Ok(t) => {
+                    r.read_unknown(bytes, t)?;
+                }
+                Err(e) => return Err(e),
+            }
+        }
+        Ok(msg)
+    }
+}
+impl<'a> MessageWrite for Data<'a> {
+    fn get_size(&self) -> usize {
+        0 + 1
```

Comment:
same as before, the `0` can be omitted

Reply:
If recall correctly, the zero was added here to force integers, but that may have been an really old version of pb-rs I've last worked on.